### PR TITLE
[1LP][RFR] Automate: test_provider_summary_topology

### PR DESCRIPTION
**Pull request by valaparthvi**
## Purpose or Intent
- __Adding tests__ 
    1. test_provider_summary_topology
- __Enhancement__ 
    1. Add `is_displayed` to `BaseTopologyView`.

### PRT Run
{{ pytest: cfme/tests/webui/test_general_ui.py -k "test_provider_summary_topology"  -sqvvv }}

### CODE REVIEW

**Comment by valaparthvi** [comment]
The tests have passed, but for some reason, PRT keeps resulting in the Invalid state.
[510 results](http://10.16.45.124/pr9871-r466-downstream-510z-RodINT/artifacts/report.html)
[511 results](http://10.16.45.124/pr9871-r466-downstream-511z-MeJdjH/artifacts/report.html)

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/webui/test_general_ui.py [hunk 1]
```diff
@@ -685,3 +687,31 @@ def test_ui_notification_icon():
...
+            2. Click on topology.
+        expectedResults:
+            1.
+            2. Provider Topology must be displayed.
+
+    Bugzilla:
+        1532404
+    """
+    view = navigate_to(provider, "Details")
+    view.entities.summary("Overview").click_at("Topology")
```
Comment (mshriver) [inline cfme/tests/webui/test_general_ui.py]: Consider defining a navigation destination for this page.

**Comment by mshriver** [comment]
PRT is false negative, tests passed fine, but because of a riggerlib deadlock the PRT processes are hanging.

**Pull request merged**